app/js/HealthCheckController.js

```diff
@@ -0,0 +1,26 @@
+const logger = require('logger-sharelatex')
+
+const MongoHandler = require('./MongoHandler')
+const ProjectHandler = require('./ProjectHandler')
+const { expressify } = require('./Utils')
+
+module.exports = {
+  status,
+  healthCheck: expressify(healthCheck)
+}
+
+function status(req, res) {
+  logger.log('hit status')
+  res.send('read-only is alive')
+}
+
+async function healthCheck(req, res) {
+  if (!MongoHandler.isConnected()) {
```

Comment:
Do you know if `isConnected` actively pings the server? (i.e. will it notice if the network drops out after an initially successful connection?) -  how about a simple query on the collections we need (maybe `db.users.count()` and `db.projects.count()`?)

Reply:
Good point. changed it to a call to db.admin().ping()